src/GovUk.Education.ExploreEducationStatistics.Data.Api/GovUk.Education.ExploreEducationStatistics.Data.Api.csproj

```diff
@@ -18,10 +18,13 @@
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.18.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.18" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.1.7" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.18" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.18" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.18" />
-    <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="3.1.18" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
Just wondering which assets are being excluded here any why?

Reply:
No idea on this one - Rider has automatically added this. I'll investigate.

Reply:
Not read and understood yet, but this seems relevant: https://docs.microsoft.com/en-us/ef/core/cli/services

Reply:
It seems like `Design` is used during development, when you use `dotnet ef` commands, for example. So I've changed csproj files, adding `PrivateAssets=all` so that anything related to it won't get published.

If this is an issue, I suspect it won't be revealed until this deploys to Dev.